Dockerfile.aarch64

```diff
@@ -15,14 +15,20 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM aarch64/ubuntu:xenial
+FROM arm64v8/debian:stretch
```

Comment:
I have no idea where `arm64v8` came from, it makes little sense. It probably is easiest if it stays, so it matches `uname`.